src/bp/ui-shared/tsconfig.json

```diff
@@ -21,6 +21,7 @@
       "~/*": ["*"],
       "botpress/sdk": ["../../sdk/botpress.d.ts"],
       "common/*": ["../../common/*"],
+      "react-select": ["../../ui-studio/node_modules/react-select"],
```

Comment:
Can you just get rid of react-select ? Prefer blueprint instead

Reply:
Yep ended up using blueprint, just forgot that there